setup.py

```diff
@@ -43,13 +44,51 @@
         )
     )
 
+
+def test_otio():
+    """Discovers and runs tests"""
+    try:
+        # Clear the environment of a preset media linker
+        del os.environ['OTIO_DEFAULT_MEDIA_LINKER']
+    except KeyError:
+        pass
+    return unittest.TestLoader().discover('tests')
+
+
 setup(
     name='OpenTimelineIO',
     version='0.8.dev',
     description='Editorial interchange format and API',
     author='Pixar Animation Studios',
     author_email='opentimelineio@pixar.com',
     url='http://opentimeline.io',
+    license='Modified Apache 2.0 License',
+
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Topic :: Multimedia :: Graphics',
+        'Topic :: Multimedia :: Video',
+        'Topic :: Multimedia :: Video :: Display',
+        'Topic :: Multimedia :: Video :: Non-Linear Editor',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'License :: Other/Proprietary License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
I'm not sure how the tagging is supposed to be set up.  Officially we support python 2.7 and 3.5, those are the  only two versions we test against.  It probably works in the versions you list, but I'm not sure how strict this is meant to be.

Reply:
I'll remove the other specified versions of 3 from the list.